plugins/BEdita/Core/config/Migrations/20240409130149_CategoriesLabels.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+
+class CategoriesLabels extends AbstractMigration
+{
+    /**
+     * @inheritDoc
+     */
+    public function up(): void
+    {
+        // create field labels
+        $this->table('categories')
+            ->addColumn('labels', 'json', [
+                'default' => null,
+                'null' => true,
+                'after' => 'name',
+            ])
+            ->update();
+        $statement = [
+            'mysql' => 'JSON_OBJECT("default", label)',
+            'mariadb' => 'JSON_OBJECT("default", label)',
+            'pgsql' => 'JSON_BUILD_OBJECT(\'default\', label)',
+            'sqlite' => 'JSON(\'{"default": "\' || label || \'"}\')',
+        ];
+        $this->query(
+            'UPDATE categories SET labels = ' . $statement[$this->getAdapter()->getAdapterType()]
+        );
+        // drop field label
+        $this->table('categories')
+            ->removeColumn('label')
+            ->update();
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function down(): void
+    {
+        // create field label
+        $this->table('categories')
+            ->addColumn('label', 'string', [
+                'default' => null,
+                'limit' => 255,
+                'null' => true,
+                'after' => 'name',
+            ])
+            ->update();
+        // copy labels.default into label
+        $this->query(
+            'UPDATE categories SET label = JSON_UNQUOTE(labels->>"$.default")'
```

Comment:
PostgreSQL: `UPDATE categories SET label = labels->>'default'`
SQLite: `UPDATE tags SET label = labels->>'$.default'`

Reply:
I didn't try the rollback on those dbs :sweat_smile: Thanks, I'll patch this soon

Reply:
done, on tag migration too